IPython/html/static/services/kernels/serialize.js

```diff
@@ -75,15 +75,17 @@ define([
         msg = _.clone(msg);
         var offsets = [];
         var buffers = [];
-        msg.buffers.map(function (buf) {
-            buffers.push(buf);
-        });
+        var i;
+        for (i = 0; i < msg.buffers.length; i++) {
+            // msg.buffers elements could be either views or ArrayBuffers
+            // buffers elements are ArrayBuffers
+            buffers.push(msg.buffers[i].buffer || msg.buffers[i])
```

Comment:
All views have a .buffer attribute, and an ArrayBuffer does not.  So it handles those cases nicely.  Not all browser support all views, so I don't want to put a hardcoded list in.  But I could do something like 
```python
b = msg.buffers[i];
buffers.push(b.buffer instanceof ArrayBuffer ? b.buffer : b);
```
if you're more comfortable with that.